notebooks/official/pipelines/pipelines_intro_kfp.ipynb

```diff
@@ -634,15 +568,15 @@
       "source": [
         "### Define a pipeline that uses the components\n",
         "\n",
-        "Next, define a pipeline that uses these three components.\n",
+        "Next, define a pipeline that uses the above three components.\n",
         "\n",
-        "By evaluating the component definitions above, you've created task factory functions that are used in the pipeline definition to create the pipeline steps.\n",
+        "By evaluating the component definitions above, you've created task factory functions that are used in the pipeline definition for creating the pipeline steps.\n",
         "\n",
         "The pipeline takes an input parameter, and passes that parameter as an argument to the first two pipeline steps (`hw_task` and `two_outputs_task`).\n",
         "\n",
         "Then, the third pipeline step (`consumer_task`) consumes the outputs of the first and second steps.  Because the `hello_world` component definition just returns one unnamed output, you refer to it as `hw_task.output`.  The `two_outputs` task returns two named outputs, which you access as `two_outputs_task.outputs[\"<output_name>\"]`.\n",
         "\n",
-        "*Note:* In the `@dsl.pipeline` decorator, you're defining the `PIPELINE_ROOT` Cloud Storage path to use.  If you had not included that info here, it would be required to specify it when creating the pipeline run, as you'll see below."
+        "**Note:** In the `@dsl.pipeline` decorator, you define `PIPELINE_ROOT` as the Cloud Storage path that is used as root folder.  You can choose to skip it now but then, you're supposed to provide it when creating the pipeline run."
```

Comment:
comma in wrong place and some edits:

- You can choose to skip it, but you have to provide it when creating the pipeline run.